### PR TITLE
Update bin reader

### PR DESCRIPTION
**Pull request by lewisjared**
# Pull request

Please confirm that this pull request has done the following:

Requires
- [x] #303 

- [x] Tests added
- [x] Documentation added (where applicable)
- [x] Example added (either to an existing notebook or as a new notebook, where applicable)
- [x] Description in ``CHANGELOG.rst`` added

### CODE REVIEW

File: pymagicc/io/binout.py [hunk 1]
```diff
@@ -147,4 +139,118 @@ def process_data(self, stream, metadata):
             "todo": ["SET"] * len(regions),
```
Comment (lewisjared) [inline pymagicc/io/binout.py:139]: @znicholls Should the dat files be SET or N/A by default? The ASCII files are N/A and the old bin reader used SET.

Comment (znicholls) [reply pymagicc/io/binout.py]: Hmm good question. I would say N/A and change the old bin reader

**Comment by znicholls** [comment]
sorry to be a pain but can we use e.g. "not_relevant" or just not include "TODO" at all (probably easier to use "not_relevant" for now for consistency with the ASCII, or also remove TODO from the OUT readers). Having "N/A" in metadata causes all sorts of issues with groupby etc. because pandas thinks it's nan.

**Comment by lewisjared** [comment]
> sorry to be a pain but can we use e.g. "not_relevant" or just not include "TODO" at all (probably easier to use "not_relevant" for now for consistency with the ASCII, or also remove TODO from the ASCII OUT readers). Having "N/A" in metadata causes all sorts of issues with groupby etc. because pandas thinks it's nan.

Had a quick look at getting rid of the TODO column for the out/binout readers, but it ended up being a reasonable change so let's leave that for another day. Changing the out and the binout reader default TODO value to "not_relevant"

**Pull request merged**